### PR TITLE
Add flag to determine whether partial success results are valid

### PR DESCRIPTION
**Pull request by Aiee**
As title

depends on https://github.com/vesoft-inc/nebula-common/pull/377

### CODE REVIEW

File: src/executor/query/GetNeighborsExecutor.h [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 #include "executor/QueryStorageExecutor.h"
 #include "planner/Query.h"
+#include "service/GraphFlags.h"
```
Comment (yixinglu) [inline src/executor/query/GetNeighborsExecutor.h:20]: Need not to include this header file here.

File: src/executor/query/IndexScanExecutor.h [hunk 2]
```diff
@@ -11,6 +11,7 @@
 #include "common/clients/storage/GraphStorageClient.h"
 #include "executor/QueryStorageExecutor.h"
 #include "planner/Query.h"
+#include "service/GraphFlags.h"
```
Comment (yixinglu) [inline src/executor/query/IndexScanExecutor.h:14]: Move this include stmt to cpp file.

File: conf/nebula-graphd.conf.default [hunk 3]
```diff
@@ -23,6 +23,10 @@
 # Copy log messages at or above this level to stderr in addition to logfiles. The numbers of severity levels INFO, WARNING, ERROR, and FATAL are 0, 1, 2, and 3, respectively.
 --stderrthreshold=2
 
+########## query ##########
+# Whether to treat partial success as an error
+--accept_partial_success=false
```
Comment (Shylock-Hg) [inline conf/nebula-graphd.conf.default:28]: Comment that it's only used for read-only access.

Comment (Shylock-Hg) [reply conf/nebula-graphd.conf.default:28]: Modify access don't accept partial success.

Comment (Aiee) [reply conf/nebula-graphd.conf.default:29]: Comments have been updated

File: src/service/QueryInstance.cpp [hunk 4]
```diff
@@ -142,6 +142,7 @@ void QueryInstance::onError(Status status) {
         case Status::Code::kIndexNotFound:
         case Status::Code::kInserted:
         case Status::Code::kKeyNotFound:
+        case Status::Code::kPartialSuccess:
```
Comment (Shylock-Hg) [inline src/service/QueryInstance.cpp:145]: When return this error?

Comment (Aiee) [reply src/service/QueryInstance.cpp:145]: Currently this error will be processed as ErrorCode::E_EXECUTION_ERROR in QueryInstance

**Review by yixinglu — APPROVED** [review]

**Review by Shylock-Hg — APPROVED** [review]

**Pull request merged**